clippy.toml

```diff
@@ -1 +1 @@
-msrv = "1.56.1"
+msrv = "1.63.0"
```

Comment:
This file should be no longer needed, `clippy` can read `Cargo.toml`

Reply:
Cool, we can remove this all over the place if that is the case.